iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -420,8 +425,14 @@ class ConversationListViewController: FullScreenViewController,
         return true
     }
 
-    func swipeableInputAccessoryDidFinishSwipe(_ view: SwipeableInputAccessoryView) {
+    func swipeableInputAccessory(_ view: SwipeableInputAccessoryView,
+                                 didFinishSwipeSendingSendable didSend: Bool) {
+        
         self.collectionView.isUserInteractionEnabled = true
+
+        if let currentCell = self.collectionView.getCentermostVisibleCell() as? ConversationMessagesCell {
+            currentCell.unprepareForNewMessage(reloadMessages: !didSend)
+        }
```

Comment:
I updated this delegate function so that you know if a message was sent after a swipe ended. If the message wasn't sent, we need to remove the placeholder message.